ext/gmp/gmp.c

```diff
@@ -893,8 +910,8 @@ int gmp_import_export_validate(zend_long size, zend_long options, int *order, in
 			*order = 1;
 			break;
 		default:
-			php_error_docref(NULL, E_WARNING,
-				"Invalid options: Conflicting word orders");
+			/* options argument is in second position */
+			zend_argument_value_error(3, "has conflicting word orders");
```

Comment:
_maybe_ something like `GMP_MSW_FIRST and GMP_LSW_FIRST cannot be defined in the same time`